src/components/ButtonSection.js

```diff
@@ -66,6 +66,9 @@ class ButtonSection extends Component {
   };
 
   handleTimeSpanClick = () => {
+    // Clear existing alertObj
+    this.clearAlert();
+
```

Comment:
When `Add a Timespan` button is clicked when there are no empty time slots to add a new timespan ahead of the current playhead, the application emits a warning. If a user moves the playhead in the waveform and click on `Add a Timespan` button again without closing the alert, it stays while the form collapses for the new timespan.
Adding this here fixes this issue.

Reply:
Good catch.